packages/accounts-passwordless/passwordless_server.js

```
@@ -235,15 +235,16 @@ Accounts.sendLoginTokenEmail = async ({ userId, sequence, email, extra = {} }) =
     sparse: true,
   });
-  Meteor.users.createIndexAsync('services.passwordless.token', {
+  await users.createIndexAsync('services.passwordless.token', {
     unique: true,
     sparse: true,
   });
 };
 
-Meteor.startup(() => setupUsersCollection());
+Meteor.startup(() => createPasswordlessIndexes(Meteor.users));
+Accounts.onUsersCollectionChanged(users => createPasswordlessIndexes(users).then());
```

ESLint

[error] Replace `·createPasswordlessIndexes(users).then()` with `⏎··createPasswordlessIndexes(users).then(),⏎`

(prettier/prettier)